qbft/spectest/tests/controller/latemsg/late_proposal_past_instance.go

```diff
@@ -10,10 +10,19 @@ import (
 
 // LateProposalPastInstance tests process proposal msg for a previously decided instance
 func LateProposalPastInstance() *tests.ControllerSpecTest {
-	identifier := types.NewMsgID(testingutils.TestingValidatorPubKey[:], types.BNRoleAttester)
 	ks := testingutils.Testing4SharesSet()
 
-	allMsgs := testingutils.DecidingMsgsForHeight([]byte{1, 2, 3, 4}, identifier[:], 5, ks)
+	allMsgs := []*qbft.SignedMessage{
```

Comment:
same comment for DecidingMsgsForHeight as before, might be better to refactor this test

Reply:
refactored